fireapp/lib/presentation/calendar/calendar_widget.dart

```diff
@@ -168,6 +173,28 @@ class _CalendarState extends FireAppState<CalendarView>
     );
   }
 
+  Widget _buildShiftCard(Shift shiftEvent, CalendarEvent calendarEvent, int index, int totalEvents) {
+    final title = shiftEvent.title;
+    final displayTime = calendarEvent.displayTime;
+    final eventId = shiftEvent.shiftId;
+    final cardColor = viewModel.getColorForEvent(eventId.toString());
+
+    return Card(
+      color: cardColor,
+      margin: const EdgeInsets.symmetric(vertical: 4, horizontal: 8),
+      child: ListTile(
+        title: Text(title),
+        subtitle: Column(
+          crossAxisAlignment: CrossAxisAlignment.start,
+          children: [
+            Text(displayTime),
+            Text('Status: ${shiftEvent.status}'),
```

Comment:
Please use translatable strings for this (there should be examples of strings with substitutions in there)